content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Uninstall Pluggable Components from KubeSphere"
+keywords: "Installer, uninstall, KubeSphere, Kubernetes"
+description: "How to uninstall pluggable components from KubeSphere"
+linkTitle: "Uninstall Pluggable Components from KubeSphere"
+Weight: 16500
+---
+
+After you [enable pluggable components of KubeSphere](../../../pluggable-components/), you can also uninstall them using the following commands. Please back up any necessary data before you uninstall them.
+
+## App Store
```

Comment:
@Sherlock113 
Considering the user's search habits, it would be better to use a full description.
Example: App Store => Uninstall KubeSphere App Store